internal/types/document_validation.go

```diff
@@ -38,6 +38,8 @@ func (e *ValidationError) Error() string {
 // ValidateData checks if the document represents a valid "data document".
 // If the document is not valid it returns *ValidationError.
 func (d *Document) ValidateData() error {
+	var idPresent bool
```

Comment:
I have a crazier idea (let's discuss it with everyone)? 

In the `Document` implementation, it is implemented and promised that if `_id` is present, it's always in the first place. With that, should we check for something like `if d.keys < 1 || d.keys[0] != `_id` instead of the current check?

Reply:
Aas agreed at a sync call, let's just add a TODO about it.

Reply:
https://github.com/FerretDB/FerretDB/pull/1278/files#diff-2039db83db8b80f633035b7515432934e98599105f1c0126289eeb7168531e06R43 ?

Reply:
LGTM!